skull-king-app/src/components/PlayerStatusCard/PlayerStatusCard.less

```diff
@@ -180,6 +180,183 @@
   }
 }
 
+.bonusSelectionContainer {
```

Comment:
Make the scroll bars for this container only visible when they are needed.

Reply:
Also, make the selection box full width

Reply:
Move the toggle button to the left and get rid of the border.

I just want "Select bonus... [arrow]" with no other UI.